views/peeps.erb

```diff
@@ -0,0 +1,22 @@
+<html>
+
+  <head>
+  </head>
+
+  <body>
+    <h1>Latest Peeps:</h1>
+    <ul>
+      <% @peep.each do |peep| %>
+      <li>
+        <%= User.get_username(peep.user_id) %>:
```

Comment:
This is a lot of work for the view. Maybe the controller of even better, the peep model could have a method that fetches username for each peep.